docs/plan/20260416-pekko-phase-kernel-typed-implement.md

```
@@ -0,0 +1,27 @@
+# pekko-phase-kernel-typed implement 計画
+
+## 概要
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)